plugins/cilium-cni/cni-install.sh

```diff
@@ -37,12 +45,40 @@ if [ -f "${CILIUM_CNI_CONF}" ]; then
 	echo "Using existing ${CILIUM_CNI_CONF}..."
 else
 	echo "Installing new ${CILIUM_CNI_CONF}..."
-	cat > ${CNI_CONF_NAME} <<EOF
+    if [ -z "${CILIUM_POLICY_ENFORCEMENT_INTERFACE}" ]; then
```

Comment:
nit: throughout this file (and other shell scripts) the newly inserted code uses tabs rather than spaces, so the indentation is messed up if you use golang or kernel-style tab widths for shell scripts. (Even here on GitHub you can see the mismatch)